.github/actions/setup-pnpm-cache/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Setup PNPM cache
+runs:
+  using: composite
+  steps:
+    - name: Install pnpm
+      uses: pnpm/action-setup@v2.2.2
+      with:
+        version: 6
```

Comment:
Should this go in the `package.json` `engines`?

I'm not across pnpm differences between v6 and v7. I just hope it's not like yarn v1 and v2...

Reply:
Apparently, the way to go is to use "packageManager" field in package.json.

> I'm not across pnpm differences between v6 and v7. I just hope it's not like yarn v1 and v2...

I don't think it's such a massive change as with yarn. But we might want to wait for #342 before merging this so we can use v7 from the beginning.